custom/ilsgateway/tanzania/handlers/soh.py

```diff
@@ -52,6 +52,7 @@ def _cleanup(s):
 class SohFormatter(Formatter):
 
     def format(self, text):
+        text = text.replace('\r', ' ').replace('\n', ' ').strip()
```

Comment:
can't we just add this line to the beginning of the custom ils handler so that it applies to all incoming messages, or do we want to preserve the newlines for some handlers?